VirtoCommerce.Storefront.Model/Cart/Services/ICartService.cs

```diff
@@ -1,12 +1,14 @@
 using System.Collections.Generic;
 using System.Threading.Tasks;
 using PagedList.Core;
+using VirtoCommerce.Storefront.Model.Contracts;
 
 namespace VirtoCommerce.Storefront.Model.Cart.Services
 {
     public interface ICartService
     {
         Task<IPagedList<ShoppingCart>> SearchCartsAsync(CartSearchCriteria criteria);
+        Task<IPagedList<CartDescriptionDto>> SearchAccountListsAsync(CartSearchCriteria criteria);
```

Comment:
Remove this method, please. Need to use `SearchCartsAsync`, as we need not only description but also Items in the ApiListController